compose-network/prometheus/prometheus.yml

```diff
@@ -0,0 +1,11 @@
+global:
+  scrape_interval: 15s
+  evaluation_interval: 15s
+
+scrape_configs:
```

Comment:
3 s is the snapshot interval of the platform. We should confuter that here, too

Reply:
Done. I also removed the `evaluation_interval`, as we're not evaluating rules in this config, it doesn't feel necessary to modify the default.